docker/cgc_metadata_to_qiime_mapping_file.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, Gregory Poore.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# ----------------------------------------------------------------------------
+import numpy as np
+import pandas as pd
+import sevenbridges as sbg
+import glob, sys, getopt, os, json, unicodedata, argparse, time
+from pandas.io.json import json_normalize
+from tqdm import tqdm # progressbar
+from tqdm import trange
+
+parser = argparse.ArgumentParser(description="""
```

Comment:
It should be relatively quick to update to click from your code, include ```import click``` together with your other libraries and here's an [example from my PR](https://github.com/biocore/tcga/pull/2/files#diff-9186f4fdc9d906eaca7baa402d356496R146) on how to use it. There's also the [click documentation](http://click.pocoo.org/5/) for other options.